kuka_kr10_support/urdf/kr10r1100sixx.xacro

```diff
@@ -4,7 +4,7 @@
   <xacro:arg name="use_mock_hardware" default="false" />
 
   <xacro:include filename="$(find kuka_kr10_support)/urdf/kr10r1100sixx_macro.xacro" />
-  <xacro:include filename="$(find kuka_resources)/urdf/common_macro.ros2_control.xacro"/>
+  <xacro:include filename="$(find kuka_ros2_control_support)/urdf/common_macro.ros2_control.xacro"/>
```

Comment:
High-level comment: this appears to go in the direction discussed in #11, but I believe what @destogl meant was for none of the `.xacro`s to contain any reference to `ros2_control` any more.

So the only package which depends on -- and adds compatibility for -- `ros2_control` would be `kuka_ros2_control_support`.

All other packages have no `exec_depend`s, nor any references to `ros2_control` any more.

Concretely: this `xacro:include` should be removed (here and in all other files). All the `xacro:arg`s which are related to `ros2_control` should also be removed. And the macro call below should also be removed.

In essence: all top-level `.xacro`s (and the underlying `_macro.xacro`s) should be reverted to how they were. `ros2_control` support is added by wrapping the `_macro.xacro`s and combining them with the files from/in `kuka_ros2_control_support`.

Reply:
Yes, exactly. We should load the description macro files of robots into ros2_control related files in the `kuka_ros2_control_support` package.

This could though make a bit tricky dependencies, but we can consider this issue later. by explicitly adding this int the user manual.